guardian-ui/README.md

```diff
@@ -0,0 +1,32 @@
+# Guardian UI
+
+## Prerequisites NPM and Yarn
+
+[NPM](https://www.npmjs.com/) - (use [nvm](https://github.com/nvm-sh/nvm))
+
+[Yarn](https://yarnpkg.com/getting-started/installhttps://classic.yarnpkg.com/lang/en/docs/install/) - `npm install --global yarn`
```

Comment:
Let's prefer to use yarn installation provided by nix shell: See [this reference](https://github.com/fedimint/fedimint/blob/master/docs/gateway.md#developing-the-gateway)